### DISCUSSION TITLE
Backup downloading code... inquiry

### DISCUSSION THREAD

**Discussion by marcustyphoon** [AprilSylph/XKit-Rewritten#167] [Tech Office]
https://github.com/AprilSylph/XKit-Rewritten/blob/4f04b941c00bb47d0c53aa8f248b80beaa75ad15/src/browser_action/render_backup.js#L45-L49

Are these appendChild/removeChild lines actually necessary? Somewhat to my surprise, it seems to work without them, but I could imagine there being a compatibility reason to include them or etc.

(I have no idea what to title this thread.)

**Comment by AprilSylph** [comment]
That's a surprise to me too! I'll test further when I get the time and remove them if they truly are unnecessary.
